elephant/surrogates.py

```diff
@@ -0,0 +1,490 @@
+# -*- coding: utf-8 -*-
+"""
+docstring goes here.
```

Comment:
In particular, I think it would be helpful to explain in what circumstances one might want to use surrogates, preferably with a reference to a review paper, book chapter or encyclopaedia article which gives more depth.